### PR TITLE
Remya/web4 about us

### PR DESCRIPTION
**Pull request by remyaabi**
Modified About us page

### CODE REVIEW

File: plugins/cubeportfolio/css/cubeportfolio.min.css [hunk 1]
```diff
@@ -8,4 +8,4 @@
  * Released under CodeCanyon License http://codecanyon.net/licenses
```
Comment (josephjoys) [inline plugins/cubeportfolio/css/cubeportfolio.min.css]: What is the change here?

File: index.html [hunk 2]
```diff
@@ -2,821 +2,1042 @@
...
+    <link href="css/owl.carousel.css" rel="stylesheet" media="screen"/>
+    <link href="css/owl.theme.css" rel="stylesheet" media="screen"/>
+    <link href="css/animate.css" rel="stylesheet"/>
+    <link href="css/style.css" rel="stylesheet">
+    <link href="css/committe.css" rel="stylesheet">
+
+    <!-- boxed bg -->
+    <link id="bodybg" href="bodybg/bg1.css" rel="stylesheet" type="text/css"/>
+    <!-- template skin -->
+    <link id="t-colors" href="color/blue.css" rel="stylesheet">
```
Comment (josephjoys) [inline index.html]: Can you revert this back to style.css, We will change the color later once all the pages are done.

File: index.html [hunk 3]
```diff
@@ -2,821 +2,1042 @@
...
-    <section id="doctor" class="home-section bg-gray paddingbot-60">
-      <div class="container marginbot-50">
-        <div class="row">
-          <div class="col-lg-8 col-lg-offset-2">
-            <div class="wow fadeInDown" data-wow-delay="0.1s">
-              <div class="section-heading text-center">
-                <h2 class="h-bold">Doctors</h2>
-                <p>Ea melius ceteros oportere quo, pri habeo viderer facilisi ei</p>
-              </div>
+    <section id="aboutus" class="home-section bg-gray paddingbot-60">
```
Comment (josephjoys) [inline index.html]: Can you just commit this changes and avoid changes other sections, we will get merge conflicts with others

File: index.html [hunk 4]
```diff
@@ -2,821 +2,1042 @@
...
+                    <p class="fontClass light"> Our church is under the administration of Malankara Jacobite Syriac
+                        Orthodox Church's American Diocese, a non-profit religious organization incorporated in the
+                        State of New York, U.S.A. St. George Jacobite Church was formed at Seattle on December 27, 2009
+                        when our diocesan metropolitan HE Mor Titus Yeldho celebrated the first Holy Qurbana. In forming
+                        this parish our goals are to hold Holy Qurbana regularly and conduct Sunday School program as
+                        well as Youth group meetings and prayer meetings to formulate a growth and understanding of our
+                        faith and foundation for the younger generation.</p>
+                </div>
+                <div class="col-lg-6">
+                    <h4 class="fontAlignCenter fontBold">History</h4>
```
Comment (josephjoys) [inline index.html]: fontAlignCenter (Camel) is not standard in css. Can we use the inbuilt class "text-center" from bootstrap

File: index.html [hunk 5]
```diff
@@ -2,821 +2,1042 @@
...
+
+                                </div>
+                                <div  class="cbp-singlePage cbp-l-grid-team-name">Moran Mor
+                                    Igantius Aphrem II</div>
+                                <div class="cbp-l-grid-team-position">Our Spiritual Leaders</div>
+                            </li>
+                            <li class="cbp-item spiritualLeader" style="height:400px;">
+                                <div class="cbp-caption cbp-singlePage">
+                                    <div class="cbp-caption-defaultWrap imgBorderCommitte">
+                                        <img src="img/committe/Aboon-Mor Baselious Thomas 1.jpg" alt="" width="100%">
```
Comment (josephjoys) [inline index.html]: Rename these files Like (1.jpg) or with short files names with no spaces

File: index.html [hunk 6]
```diff
@@ -2,821 +2,1042 @@
...
+                                    </div>
+
+                                </div>
+                                <div class="cbp-singlePage cbp-l-grid-team-name">His Beatitude
+                                    Aboon Mor Baselios Thomas I</div>
+                                <div class="cbp-l-grid-team-position">Our Spiritual Leaders</div>
+                            </li>
+                            <li class="cbp-item spiritualLeader" style="height:400px;width:340px;">
```
Comment (josephjoys) [inline index.html]: style="height:400px;width:340px;" this will hit problems in small devices. Try to avoid inline styling.

File: css/style.css [hunk 7]
```diff
@@ -140,6 +140,12 @@ p {
 
 }
 
+img{
+
+	height:auto;
+	width: 100%;
+}
+.imgBorderCommitte
```
Comment (josephjoys) [inline css/style.css]: imgBorderCommitte is not standard in css change it to image-border-committee

File: index.html (hunk 5 above)
Comment (divpaul2016) [reply index.html]: Explicit naming is the right approach

File: index.html (hunk 2 above)
Comment (remyaabi) [reply index.html]: reverted

File: css/style.css (hunk 7 above)
Comment (remyaabi) [reply css/style.css]: updated

File: plugins/cubeportfolio/css/cubeportfolio.min.css (hunk 1 above)
Comment (remyaabi) [reply plugins/cubeportfolio/css/cubeportfolio.min.css]: reverted back the changes

File: index.html (hunk 3 above)
Comment (remyaabi) [reply index.html]: modified

File: index.html (hunk 5 above)
Comment (remyaabi) [reply index.html]: Used explicit naming and removed the spaces

File: css/committe.css [hunk 8]
```diff
@@ -0,0 +1,45 @@
+.image-border-committee{
```
Comment (divpaul2016) [inline css/committe.css]: rename to Boardoftrustee

File: doctors/member1.html [hunk 9]
```diff
@@ -1,8 +1,8 @@
 <div class="cbp-l-member-img">
```
Comment (divpaul2016) [inline doctors/member1.html]: Rename the file name to Appropriate section

Comment (josephjoys) [reply doctors/member1.html]: Are we planning showing this page ?

File: css/committe.css (hunk 8 above)
Comment (remyaabi) [reply css/committe.css]: modified

File: doctors/member1.html (hunk 9 above)
Comment (remyaabi) [reply doctors/member1.html]: @divpaul2016 @josephjoys : We decided not to use that html under doctors folder.

**Pull request merged**